docs/asyncapi.yaml

```diff
@@ -469,7 +404,7 @@ components:
       type: string
       format: DD-MM-YYYY
       description: Date SITI Agri exported payment file
-      example: "2022-06-28"
+      example: '2022-06-28'
```

Comment:
Do dates need the single quotes for it to appear properly in the example message?

Reply:
Seems not.   Changed